src/commands/CountryCommand.ts

```diff
@@ -0,0 +1,393 @@
+import { Command } from "../Command";
+import { CommandInteraction, MessageEmbed } from "discord.js";
+import {
+	inlineCode,
+	SlashCommandBuilder,
+	SlashCommandStringOption,
+	SlashCommandSubcommandsOnlyBuilder,
+} from "@discordjs/builders";
+import {
+	Country,
+	countryData,
+	CountryKey,
+	getCountryByName,
+	getFilteredCountryDataBy,
+	sortCountryDataBy,
+	typeOfCountryProperty,
+	updateDataFromSource,
+} from "../util/countryUtil/dataManager";
+
+export let formatNumber: (n: number) => string;
+
+class CountryCommand extends Command {
+	constructor() {
+		super("country");
+	}
+
+	async execute(interaction: CommandInteraction): Promise<void> {
+		formatNumber = (n: number) => {
+			return n.toLocaleString(interaction.locale);
+		};
+
+		switch (interaction.options.getSubcommand()) {
+			case "update-data": {
+				updateDataFromSource();
+				interaction.reply({ content: "On it!", ephemeral: false });
+				break;
+			}
+			case "overview": {
+				const country: Country | undefined = getCountryByName(interaction.options.getString("country") ?? "Bhutan");
```

Comment:
1. Bhutan is cool an more people should now about it
2. No it will not - the `?? Bhutan` is for the string option but not for the case when the country you entered isn't listed